result/result.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Rohan Mathur" />
+    <meta name="description" content="Evolt a quiz app for testing your knowledge" />
+
+    <!-- links -->
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../quiz/quiz.css">
+    <link rel="stylesheet" href="./result.css">
+    <title>Evolt | Results 🥳</title>
+</head>
+
+<body>
+    <nav class="navbar flex-row content-space-between items-center fixed top-0 z-1 w-100  py-2 px-6">
+        <a class="text-xl font-black" href="../index.html">Evolt</a>
+        <a class="login-cta transition-2 py-1 px-2 rounded-sm" href="../auth/login.html">login</a>
+    </nav>
+
+    <div class="flex-column items-center mt-8 mx-2">
+        <h3 class="mt-4 text-xl font-bold underline">Results</h3>
+        <h3 class="mt-4 font-bold">Final score: 8 / 10</h3>
+        <div class="content flex-column items-center">
+            <div class="quiz text-center mb-5">
+                <h3 class="my-5">Sheldon is which type of scientist?</h3>
+                <div class="quiz-options">
+                    <div class="result text-base correct-answer transition-3 font-semibold p-2 rounded-sm mb-2">

```

Comment:
why is there a question on result page ?

Reply:
For each question, which is correct answer and which is not that user should know.